athlinks-schema.ttl

```diff
@@ -145,10 +161,48 @@ athlinks:bib
 
 athlinks:pace
   a owl:DataTypeProperty ;
-  rdfs:range schema:DateTime .
+  rdfs:range schema:Duration .
 
 athlinks:seriesType
   a owl:DataTypeProperty;
-  rdfs:range[ a xsd:string ;
-		owl:oneOf("Master", "Social", "Points", "Series")
-	     ] .
+  rdfs:range [ a xsd:string ;
+								owl:oneOf("Master", "Social", "Points", "Series")
+	     				] .
+
+# For  AthleticRace PROPERTIES
+athlinks:giveaway
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:require_phone
```

Comment:
ya, I'm just figuring when we start to map Ctlive data... we are going to have tons of Registration configuration stuff...
And we likely will want to keep that separate from the Race just due to quantity / complexity of the registration configuration.